package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-device-info",
-  "version": "8.7.1",
+  "version": "9.0.0",
```

Comment:
I need you to revert this change - we have a github action using the semantic-release toolkit and it will analyze the commits (I'll make sure it's correct...) to determine what sort of release to do then it controls the version number change
https://github.com/react-native-device-info/react-native-device-info/blob/master/.github/workflows/release.yml

Reply:
netinfo is the same by the way, all automagic

Reply:
Ah I misunderstood the "Clearly document, bump the version, move on"  as you wanted me to bump it.
Will revert!

Reply:
Reverted!

Reply:
That was kind of ambiguous, sorry - thanks for the patchup, let me review and let's get this out